docs/CODE_OF_CONDUCT.md

```diff
@@ -8,19 +8,19 @@ In the interest of fostering an open and welcoming environment, we as contributo
 
 Examples of behavior that contributes to creating a positive environment include:
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+- Using welcoming and inclusive language
```

Comment:
why this change? are dashes better than asterisks?

Reply:
The standard for a bulleted list using mark down is the dash. Asterisk is preferred for the bold words. However, the file is rendered as your previous.